waitress/app/views.py

```diff
@@ -52,4 +55,35 @@ class Dashboard(LoginRequiredMixin, View):
     login_url = "/"
 
     def get(self, request):
-        return render(request, "dashboard.html")
+        users_query_set = SlackUser.objects.order_by("id")
+        users_list = users_query_set.all()
+        active_users_count = users_query_set.filter(is_active=True).count()
```

Comment:
I'm not sure if to handle pagination on the client or on the server. Curious as to your thoughts on this - @abiodun0 @osule

Reply:
It's fine handling it here in the server.  
`active_user_count` and `inactive_users_count` can be reduced to a single database query.
If you get one of them, you can figure the rest by subtracting from the count of users

Reply:
+1 to what femi said, since you are not doing any form of SPA, it's better it's on the server